dev/tasks/conda-recipes/r-arrow/configure.win

```diff
@@ -2,7 +2,7 @@
 
 set -euxo pipefail
 
-echo "PKG_CPPFLAGS=-DNDEBUG -I\"${LIBRARY_PREFIX}/include\" -I\"${PREFIX}/include\" -DARROW_R_WITH_ARROW -DARROW_R_WITH_S3" > src/Makevars.win
+echo "PKG_CPPFLAGS=-DNDEBUG -I\"${LIBRARY_PREFIX}/include\" -I\"${PREFIX}/include\" -DARROW_R_WITH_ARROW -DARROW_R_WITH_PARQUET -DARROW_R_WITH_DATASET -DARROW_R_WITH_S3" > src/Makevars.win
```

Comment:
@nealrichardson is this another change that we will need to upstream at the next release?

Reply:
Yes. You could make the change now in fact, nothing would be hurt by having extra -D flags on. https://github.com/conda-forge/r-arrow-feedstock (FYI @xhochy)

Reply:
Done https://github.com/conda-forge/r-arrow-feedstock/pull/30

Reply:
Merged in https://github.com/conda-forge/r-arrow-feedstock/commit/ed01e9badf1cd831d5201d6f9b35dcf01c873cf6